tcms/signals.py

```diff
@@ -217,14 +218,49 @@ def handle_emails_post_bug_save(sender, instance, created=False, **kwargs):
     from tcms.core.utils.mailto import mailto
 
     mailto(
-        template_name="email/post_bug_save/email.txt",
+        template_name="email/post_bug_save/email_create.txt",
         recipients=[instance.assignee.email],
         subject=_("NEW: Bug #%(pk)d - %(summary)s")
         % {"pk": instance.pk, "summary": instance.summary},
         context={"bug": instance},
     )
 
 
+def handle_emails_post_bug_comment_added(sender, instance, created=False, **kwargs):
+    """
+    Send email updates to assignee and reporter if there is a
+    new comment.
+    """
+    from tcms.core.helpers.comments import get_comments
+
+    comments = get_comments(instance)
+
+    if created or comments.count() <= 1:
+        return
+
+    last_comment = comments.last()
```

Comment:
You can merge this line and the mailto function inside `handle_emails_post_bug_save`. There isn't much difference between that function and the one you are adding so they may as well be in the same place. 

See some of the other post_save handlers for examples of new vs. update emails.